GraphRag/knowledgeGraph.py

```diff
@@ -0,0 +1,54 @@
+from llama_index.core import StorageContext
```

Comment:
Explain purpose of each file as a docstring at the top of each python file.

Reply:
added docstring for each file